GameData/RealismOverhaul/RO_RecommendedMods/Procedurals/RO_ProceduralParts.cfg

```diff
@@ -460,8 +559,10 @@
 		utilizationTweakable = true
 		minUtilization = 99
 		utilization = 100
-		type = Tank-III-Balloon
-		typeAvailable = Tank-III-Balloon
+		type = Tank-Balloon-SteelAl
+		typeAvailable = Tank-Balloon-SteelAl
+		typeAvailable = Tank-Balloon-SteelAlCu
+		typeAvailable = Tank-Balloon-SteelAlLi
```

Comment:
You hijacked the existing balloon tank part which means that vessels from old saves will be broken.

Reply:
Fixed